UPGRADE.md

```diff
@@ -110,7 +110,7 @@ This should be mostly transparent for you unless:
 
 ## Changed Signature Of Data Provider Hooks
 
-Specialized data provider hooks (like `useGetList` and `useUpdate`) have a new signature. There are 2 changes:
+Specialized data provider hooks (like `useGetOne`, `useGetList`, `useGetMany` and `useUpdate`) have a new signature. There are 2 changes:
```

Comment:
```suggestion
Specialized data provider hooks (like `useGetList`, `useGetMany`, `useGetManyReference`, `useGetOne` and `useUpdate`) have a new signature. There are 2 changes:
```

Reply:
I've added `useGetManyReference` here, is simpler than reviewing the same in the other PR